### PR TITLE
UFC test cases

### PR DESCRIPTION
**Pull request by schmit**
This PR adds test cases for the UFC to this repo so we can use them to test our SDKs

### CODE REVIEW

File: ufc/flags-v1.json [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+                }
+              ],
+              "extraLogging": {
+                "allocationvalue_type": "rollout",
+                "owner": "Alice"
+              }
+            }
+          ],
+          "doLog": true
+        }
```
Comment (giorgiomartini0) [inline ufc/flags-v1.json:157]: I'm confused by this allocation.
1. Is it intended to be below the experiment? I could understand this to mean "we rolled out blue as the new control group to NA, and now we're running an experiment" ... but as the experiment traffic increases, more and more NA subjects will actually end up getting the _previous_ `control`?
2. Why is the end 8000? In what sense is this a rollout if it lets 20% of traffic pass through?
3. Should we have a default allocation after it? Everyone that passes through is (implicitly) getting `null` at the moment. We should enforce a default allocation at UFC generation time. (If you want to make sure the SDK is robust to that not working as intended, it should be a dedicated test.)

Comment (schmit) [reply ufc/flags-v1.json:157]: The goal here is to test all functionality of the UFC, so the config might not make the most sense in practice.

File: ufc/flags-v1.json [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+          "key": "experiment",
+          "rules": [],
+          "startAt": null,
+          "endAt": null,
+          "splits": [
+            {
+              "variationKey": "control",
+              "shards": [
+                {
+                  "salt": "traffic-new-user-onboarding-experiment",
```
[the remark this answers is not in the document]
Comment (schmit) [reply ufc/flags-v1.json:88]: Going to leave as is as the formatting here does not matter for testing purposes.

**Review by giorgiomartini0 — APPROVED** [review]
I didn't review the test case assignments line-by-line. Structure of `flags-v1.json` and overall coverage looks good!

**Pull request merged**